devfiles/apache-camel-springboot-che7.yaml

```diff
@@ -0,0 +1,56 @@
+---
+displayName: Apache Camel based projects on Che 7
+description: Stack with environment ready to develop Integration projects with Apache Camel based on SpringBoot.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+specVersion: 0.0.1
+name: apache-camel-springboot-che7
+components:
+  -
+    type: cheEditor
+    id: eclipse/che-theia/next
+    memoryLimit: 512Mi
```

Comment:
Can we avoid adding theia and the exec plugin?

Reply:
We can but I'm not sure if we should.

I mean, as about exec plugin - I guess we can and should remove it from Devfiles, will do it.
But about Theia, it would make Devfiles smaller (- 3 lines) but at the same time:
1. Memory limit could not be overridden for theia in such case (not sure if it's needed, just did the same as Che 7 Stack has).
2. Devfile contains plugins which are not compatible with any other editor except Theia. Just theoretical case: what if Che Server is configured with `gwt`/`jupyter` editor, and devfile contains plugins for Theia but Theia is not declared.
@l0rd Do you think we should remove theia as well?

Reply:
@sleshchenko yes I believe we should remove it. The day we will have real world `gwt/jupyter` use cases we will think about it (there will be probably a a dedicated devfile registry).

Reply:
OK will do it for stacks where memory limit for theia is not needed, will ask authors

Reply:
@l0rd In this issue `512m` is recommended to use for Theia https://github.com/eclipse/che/issues/12620
But default value in Plugin Registry is `1536M` https://che-plugin-registry.openshift.io/v3/plugins/eclipse/che-theia/next/

Should I create an issue to make default memory limit for Theia 512, 
or keep theia component in these devfiles to set 512 memory limit?

Reply:
Yes it should be 512m by default. Don't know why we still have the old value but it must be changes.

Reply:
The corresponding PR is created https://github.com/eclipse/che-plugin-registry/pull/156